packages/stark-ui/src/modules/route-search/components/route-search.component.spec.ts

```diff
@@ -0,0 +1,153 @@
+/* tslint:disable:completed-docs no-commented-code prefer-const*/
```

Comment:
Why is the `prefer-const` rule disabled? This should never be disabled since it is really easy to fix 😉